pkg/apis/cluster/v1alpha1/cluster_types.go

```diff
@@ -136,6 +136,9 @@ type ClusterStatus struct {
 	// GitVersion of the kubernetes cluster, this field is populated by cluster controller
 	KubernetesVersion string `json:"kubernetesVersion,omitempty"`
 
+	// GitVersion of the /kapis/version api response, this field is populated by cluster controller
+	KubeSphereVersion string `json:"kubesphereVersion,omitempty"`
```

Comment:
letter S should be capitalized.

`kubesphereVersion` to `kubeSphereVersion`

Reply:
Thanks for your suggestion. I've pushed code.